packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1548,6 +1560,9 @@ class RenderPhysicalModel extends _RenderPhysicalModelBase<RRect> {
     _markNeedsClip();
   }
 
+  /// Whether and how to clip.
```

Comment:
by the way if you come up with detailed docs for these and want to avoid copying-and-pasting them all over the place, and if they do in fact apply equally everywhere, then consider using a dartdoc macro/template to avoid duplication. See the child macro (used for the `child` property of most widgets) as an example.

Reply:
Done.